examples/anthos-service-mesh-multicluster/postgres/certs/client-cert.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
certs should not be checked into VCS. you already provide instructions to download these from cloud sql
Please add to .gitignore.

Reply:
Certs removed and *.pem added to .gitignore